src/utils/AutoDiscoveryUtils.js

```diff
@@ -34,6 +34,8 @@ export class ValidatedServerConfig {
     isUrl: string;
 
     isDefault: boolean;
+    // when the server config is based on static URLs the hsName is not resolvable and things may wish to use hsUrl
+    static: boolean;
```

Comment:
Hmm, this meaning of `static` seems a bit confusing, and I've already come back to here to read it 3 times during the review... Maybe flip the sense around to `isNameResolvable`? Anyway, up to you.

Reply:
Naming is hard, your suggestion is much better than the mind blank I had when I wrote `static` :D